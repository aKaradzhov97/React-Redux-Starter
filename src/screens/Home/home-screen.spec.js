import React from 'react';
import { render, screen } from '../../tests/test-utils';
import HomeScreen from './home-screen';

describe('HomeScreen', () => {
  let initialState;

  beforeAll(() => {
    initialState = {};
  });

  describe('Rendering', () => {
    it('should render the component with initial state', () => {
      const { container } = render(<HomeScreen />, { initialState });

      expect(container.querySelector('body section')).toMatchSnapshot();
    });
  });
});
