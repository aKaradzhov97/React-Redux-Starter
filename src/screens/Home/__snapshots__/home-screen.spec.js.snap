// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen Rendering should render the component with initial state 1`] = `
<section>
  <h1>
    home_screen.title
  </h1>
  <a
    href="/about"
  >
    home_screen.go_about_link
  </a>
</section>
`;
